client/src/components/Ticketing/checkout/CheckoutSuccess.tsx

```diff
@@ -10,28 +10,46 @@
 import React, {ReactElement} from 'react';
 import {useNavigate} from 'react-router-dom';
 
+interface CheckoutSuccessProp {
+  completeOrderButton: string;
+}
+
 /**
  * Displays the CheckoutSuccess page
- *
+ * @param CheckoutSuccessProp
+ * @param CheckoutSuccessProp.completeOrderButton
  * @returns {ReactElement}
  */
 export default function CheckoutSuccess(): ReactElement {
   const navigate = useNavigate();
 
   return (
     <div className='w-full h-screen items-center flex flex-row bg-zinc-100'>
-      <div className="w-full flex flex-col items-center text-2xl font-bold text-zinc-700 ">
+      <div className='w-full flex flex-col items-center text-2xl font-bold text-zinc-700 '>
         <div className='flex flex-col items-center gap-5 bg-white p-7 rounded-xl shadow-xl'>
-          <div className="flex flex-row items-center ">
-            <svg xmlns="http://www.w3.org/2000/svg" className="h-12 w-12" viewBox="0 0 20 20" fill="currentColor">
-              <path fillRule="evenodd" d="M6.267 3.455a3.066 3.066 0 001.745-.723 3.066 3.066 0 013.976 0 3.066 3.066 0 001.745.723 3.066 3.066 0 012.812 2.812c.051.643.304 1.254.723 1.745a3.066 3.066 0 010 3.976 3.066 3.066 0 00-.723 1.745 3.066 3.066 0 01-2.812 2.812 3.066 3.066 0 00-1.745.723 3.066 3.066 0 01-3.976 0 3.066 3.066 0 00-1.745-.723 3.066 3.066 0 01-2.812-2.812 3.066 3.066 0 00-.723-1.745 3.066 3.066 0 010-3.976 3.066 3.066 0 00.723-1.745 3.066 3.066 0 012.812-2.812zm7.44 5.252a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z" clipRule="evenodd" />
+          <div className='flex flex-row items-center '>
+            <svg
+              xmlns='http://www.w3.org/2000/svg'
+              className='h-12 w-12'
+              viewBox='0 0 20 20'
+              fill='currentColor'
+            >
+              <path
+                fillRule='evenodd'
+                d='M6.267 3.455a3.066 3.066 0 001.745-.723 3.066 3.066 0 013.976 0 3.066 3.066 0 001.745.723 3.066 3.066 0 012.812 2.812c.051.643.304 1.254.723 1.745a3.066 3.066 0 010 3.976 3.066 3.066 0 00-.723 1.745 3.066 3.066 0 01-2.812 2.812 3.066 3.066 0 00-1.745.723 3.066 3.066 0 01-3.976 0 3.066 3.066 0 00-1.745-.723 3.066 3.066 0 01-2.812-2.812 3.066 3.066 0 00-.723-1.745 3.066 3.066 0 010-3.976 3.066 3.066 0 00.723-1.745 3.066 3.066 0 012.812-2.812zm7.44 5.252a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z'
+                clipRule='evenodd'
+              />
             </svg>
-      Thank you for your purchase!
+            Thank you for your purchase!
           </div>
-          <button onClick={() => navigate('/')} className="bg-blue-500 px-5 py-3 text-white text-lg font-normal rounded-xl hover:bg-blue-600"> Back to Home</button>
+          <button
+            onClick={() => navigate('/')}
+            className='bg-blue-500 px-5 py-3 text-white text-lg font-normal rounded-xl hover:bg-blue-600'
+          >
+            {' '}
```

Comment:
Instead of adding a space here, you could add a margin-right to the `div` element just before the `button`

Reply:
Removed the space.